### DISCUSSION TITLE
Insuline sensitivity learning

### DISCUSSION THREAD

**Discussion by dxnxz** [NightscoutFoundation/xDrip#2955] [Ideas]
Hey guys, I just wondered if it wouldn't be nice to have xDrip learn about the insuline sensitivity over time. Currently the app calculates the forecast based on the sensitivities i set up. My idea would be that xDrip suggests (or offers) an additional forecast line for the change of glucose based on historical data of the last 2 weeks, let's say. What do you guys think about that?

**Comment by Navid200** [comment]
This may be related.  
https://github.com/NightscoutFoundation/xDrip/discussions/2651
